### ISSUE TITLE
Rule proposal: combine-imports

### ISSUE THREAD

**Issue by danielmoore** [eslint/eslint#3478]
I think it'd be nice to have a stylistic rule for ES2015 code that considers multiple imports from the same module as a violation.

So, this would be considered a violation:

```js
import { join as joinPath } from 'path';
import { readdir } from 'fs';
import { resolve as resolvePath } from 'path';
```

and these would not:

```js
import { join as joinPath, resolve as resolvePath } from 'path';
import { readdir } from 'fs';
```

```js
import path, { join as joinPath, resolve as resolvePath } from 'path';
import { readdir } from 'fs';
```

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by xjamundx** [comment]
If this doesnt' make it into core I'm happy to add this to thew new plugin I'm working on that hopes to capture various rules around js modules:
https://github.com/xjamundx/eslint-plugin-modules

**Comment by nzakas** [comment]
I think it makes sense in core, similar to `one-var`.

**Comment by IanVS** [comment]
This rule seems to already exist as eslint-plugin-import/no-duplicates (https://www.npmjs.com/package/eslint-plugin-import). Is it still worth including in core?

**Comment by nzakas** [comment]
There hasn't been any movement here in the past seven months, so might make sense to close and refer people to the plugin.

**Comment by SimenB** [comment]
@nzakas #5455
